public/js/sb-admin-2.js

```diff
@@ -0,0 +1,56 @@
+(function($) {
+  "use strict"; // Start of use strict
+
+  // Toggle the side navigation
+  $("#sidebarToggle, #sidebarToggleTop").on("click", function(e) {
+    $("body").toggleClass("sidebar-toggled");
+    $(".sidebar").toggleClass("toggled");
+    if ($(".sidebar").hasClass("toggled")) {
+      $('.sidebar .collapse').collapse("hide");
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mamoon97150/Blog-PHP/pullRequest?prid=7234961)
```suggestion
      $(".sidebar .collapse").collapse("hide");
```